### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by CzerPDX**
In the main function a while loop is invoked that runs until the variable "num" is greater than 100.
Next, processFizzbuzz runs. This is a function that aims to build an output string based on the value of an input number "num". It does the following:

- First it checks if num % 3 is 0. If so, it concatenates "fizz" to the output string.
- Then it checks if num % 5 is 0. If so, it concatenates "buzz" to the output string.
- Then it checks if the string is still empty. If it is, it sets output to be the incoming number.
- Then it returns the output

In the main function it then prints the returned output to the console using console.log()
Then it increments num and restarts the loop

I did my best to follow the airbnb javascript style guide, but there's a lot in there so please let me know if you notice that I've missed something! I'm open to all feedback on my work!

### CODE REVIEW

**Review by gleason9113 — APPROVED** [review]
Code is well-formatted and readable.
Produces expected output when run.
Review passes.

**Review by Saylee119 — APPROVED** [review]
Your code looks good. I am approving your PR

File: hw1/01-fizzbuzz.js [hunk 1]
```diff
@@ -8,9 +8,27 @@ Write a program that writes all the numbers from 1 to 100, with some exceptions:
 Use console.log() to write the proper output to the command line.
 
 **/
+function processFizzbuzz(num) {
+  let output = ''
+  if ((num % 3) === 0) {
```
Comment (smah29) [inline hw1/01-fizzbuzz.js]: logic is correct, but it is complex to uderstand at first glance and I think using if, else if and else statement is better than using multiple if statements

**Comment by JeremyIV** [comment]
Has this most recent version been tested? On line 20, `nums` is undefined, correct? So for all numbers not divisbile by 3 or 5, I would expect this code to log `undefined`

**Review by Shweta-Tapele — APPROVED** [review]
Code is Easy to Understand. 
Produces output.
Approving your PR

**Comment by CzerPDX** [comment]
> Has this most recent version been tested? On line 20, `nums` is undefined, correct? So for all numbers not divisbile by 3 or 5, I would expect this code to log `undefined`

Ah, yes this is correct. I think I accidentally entered that 's' on the end when compulsively saving it before pushing! Thank you for catching that, since I added it the error after completing my last test!

**Pull request merged**